plugins/modules/zabbix_host.py

```diff
@@ -966,9 +966,29 @@ def update_exist_interfaces_with_defaults(exist_interfaces):
         "privprotocol": 0,
         "privpassphrase": ""
     }
+    allowed_fields = [
+        'hostid',
+        'type',
+        'ip',
+        'dns',
+        'port',
+        'useip',
+        'main',
+        'details',
+        'interface_ref',
+        'items',
+        'interfaceid'
+    ]
     for interface in exist_interfaces:
+        normalized_interface = interface
+        if LooseVersion("7.0") <= LooseVersion(zabbix_version):
```

Comment:
Shouldn't this be >= ?  6.4 and 6.0 were working fine before.

Reply:
Is it wrong code? checking fields  will runs for jost 7.0 and higher.

Reply:
Right now you're checking for <= 7.0.  That means it won't run in the future for 7.2, but it will run today for 6.4.  6.4 is working fine (don't think it needs the fields, but the API handles them gracefully).  Either it needs to be >= 7.0 or we just sanitize them for all versions (which I don't honestly think is necessarily wrong).

Reply:
> Right now you're checking for <= 7.0

Does this mean integration test target is 7.0 and lower? 

>  if LooseVersion("7.0") <= LooseVersion(zabbix_version):

If 6.4 and 6.0, zabbix_host module's behavior is not chenged from current behavior.  in addition, this effects on 7.0, 7.2 and higher.


> Either it needs to be >= 7.0 or we just sanitize them for all versions (which I don't honestly think is necessarily wrong).

If you want to check for all version, I will remove version checking condition.